homeassistant/components/transmission.py

```diff
@@ -50,20 +53,22 @@
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
         vol.Optional(TURTLE_MODE, default=False): cv.boolean,
+        vol.Optional(CONF_SCAN_INTERVAL, default=SCAN_INTERVAL):
+            cv.time_period,
         vol.Optional(CONF_MONITORED_CONDITIONS, default=['current_status']):
         vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
     })
 }, extra=vol.ALLOW_EXTRA)
 
-SCAN_INTERVAL = timedelta(minutes=2)
-
 
 def setup(hass, config):
     """Set up the Transmission Component."""
-    host = config[DOMAIN][CONF_HOST]
-    username = config[DOMAIN][CONF_USERNAME]
-    password = config[DOMAIN][CONF_PASSWORD]
-    port = config[DOMAIN][CONF_PORT]
+    conf = config['m'+DOMAIN]
```

Comment:
what's with the 'm' ?

Reply:
My fault, while merging with my custom component.